cmd/livepeer/livepeer.go

```diff
@@ -54,11 +54,11 @@ func main() {
 		glog.Fatalf("Cannot find current user: %v", err)
 	}
 
-	httpPort := flag.String("http", "8935", "http port")
 	rtmpPort := flag.String("rtmp", "1935", "rtmp port")
 	datadir := flag.String("datadir", fmt.Sprintf("%v/.lpData", usr.HomeDir), "data directory")
-	rtmpIP := flag.String("rtmpIP", "127.0.0.1", "IP to bind for HTTP RPC commands")
-	httpIP := flag.String("httpIP", "127.0.0.1", "IP to bind for HTTP RPC commands")
+	rtmpIP := flag.String("rtmpIP", "127.0.0.1", "IP to bind for RTMP commands")
+	cliAddr := flag.String("cliAddr", "127.0.0.1:8935", "Address to bind for  CLI commands")
+	httpAddr := flag.String("httpAddr", "", "Address to bind for HTTP commands")
```

Comment:
Those won't always be the same.

* The serviceAddr indicates the ip:port that is stored on the blockchain. This must match what is on the blockchain. (This is a 'soft must' for now.) If unset, the default is behavior to auto-discover the node's current public IP and check that against the blockchain.  There's a bunch of reasons someone would want to set the serviceAddr: if the serviceURI is a hostname, if running a private livepeer network where the serviceURI is non-public, etc. This option is really a sanity check to ensure that the transcoder's ServiceURI expectations match what is actually on the blockchain. If there's an unhandled mismatch, then it's likely the transcoder would not be able to actually receive any jobs.
* If the httpAddr is unset, listen to the service port that is stored on-chain.
* However, if the orchestrator's listening interface needs to differ from the serviceURI, then httpAddr is set. For example, if the node traffic is port-forwarded or load-balanced. So httpAddr is set in order to configure the listening interface.
* Broadcaster still needs a `-httpAddr` option since `-serviceAddr` doesn't have any meaning for them.

Most transcoders will not need to set either of these options, at least early on when things are still simple.

Writing this up I think I found an issue with the code, so will investigate that in the AM.